react/components/BrandTranslation/BrandImportModal.tsx

```diff
@@ -0,0 +1,321 @@
+import React, { useState } from 'react'
+import { useMutation, useQuery } from 'react-apollo'
+import {
+  ModalDialog,
+  ButtonPlain,
+  Dropzone,
+  Tabs,
+  Tab
+} from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+
+import { sanitizeImportJSON, parseXLSToJSON, parseJSONToXLS } from '../../utils'
+import { useLocaleSelector } from '../LocaleSelector'
+import WarningAndErrorsImportModal from '../WarningAndErrorsImportModal'
+import UPLOAD_BRAND_TRANSLATION from '../../graphql/uploadBrandTranslation.gql'
+import UPLOAD_BRAND_REQUESTS from '../../graphql/brandUploadRequests.gql'
+import ImportStatusList from '../ImportStatusList'
+
+const categoryHeaders: Array<keyof Brand> = [
```

Comment:
I think `brandHeaders` would be better here (it's wrong on product 😅)

Reply:
done, i have also applied the change in ProductImportModal